src/assignment3/Dictionary.java

```diff
@@ -0,0 +1,80 @@
+package assignment3;
+
+import java.util.List;
+
+public class Dictionary {

```

Comment:
You don't have to submit the `Dictionary` class, it is just for testing purposes.